### ISSUE TITLE
Custom delta in session replay

### ISSUE THREAD

**Issue by ghost** [aker-gateway/Aker#56]
Sometimes there are huge gaps when nothing is happening during session replay.
Lets add new flag `--delta` or `--delay` so user can specify delay in seconds between steps during `akerctl --replay`.

**Comment by anazmy** [comment]
I have this in mind:
* Define playback speed like 2X, 3X etc..
* While playing back right arrow fast forwards 10 secs, left arrow rewinds 3 secs (might need little more thinking here)
* While playing back pressing space pause/resume the playback

**Comment by ghost** [comment]
Aha, just like media player. Cool, I like it:)
We could bind Up and Down arrows to define playback speed - no parameter needed but we could have it anyway - maybe somebody will find it useful.

**Comment by anazmy** [comment]
Yeah could be that too, just need to think it through, specially making the akerctl aware of time in 2x speed

**Comment by ghost** [comment]
See this: https://github.com/g----/Aker/blob/replay-delay/aker/commands.py#L58
It is just quick version supporting `--delay` flag. But increasing speed means just using multiplier instead of constant.

Bigger problem is to actually bind keystrokes and change variable values in that for loop. That will take some coding:) But I have some idea.

**Comment by anazmy** [comment]
Yes I noticed that one ;-) thanks for that. 

I gotta wrap up some work and work on these stuff along with scale testing
